Doc Request: Cannot get "-collector.process.processes-where" flag to work

Hi,

Would it be possible to have a filter example for the "-collector.process.processes-where" flag? I can't seem to get it to work.

I get the following errors when I open localhost:9182/metrics

time="2017-08-09T16:32:16-04:00" level=error msg="[ERROR] failed collecting process metrics: <nil> Exception occurred. (Invalid query )" source="process.go:142"

time="2017-08-09T16:32:16-04:00" level=error msg="ERROR: process collector failed after 0.008049s: Exception occurred. (Invalid query )" source="exporter.go:90"

Thank you!

--- a/README.md
+++ b/README.md
@@ -60,6 +60,14 @@ See [open issues](https://github.com/martinlindhe/wmi_exporter/issues)
 The prometheus metrics will be exposed on [localhost:9182](http://localhost:9182)
 
 
+## Examples
+
+Please note: The quotes in the parameter names are required because of how Powershell parses command line arguments.
+
+### Enable only process collector and specify a custom query
+
+    .\wmi_exporter.exe "-collectors.enabled" "process" "-collector.process.processes-where" "Name='firefox'"
+
 ## License
 
 Under [MIT](LICENSE)

